packages/adena-module/src/utils/messages.ts

```diff
@@ -1,5 +1,5 @@
 import { Any, PubKeySecp256k1, Tx, TxFee, TxSignature } from '@gnolang/tm2-js-client';
-import { MsgCall, MsgAddPackage, MsgSend, MsgEndpoint } from '@gnolang/gno-js-client';
+import { MsgCall, MsgAddPackage, MsgSend, MsgEndpoint, MsgNoop } from '@gnolang/gno-js-client';
```

Comment:
Is there a link to see what `MsgNoop` looks like?

Reply:
https://github.com/gnolang/gno/issues/2209

Reply:
@jinoosss  we have PRs [2209](https://github.com/gnolang/gno/pull/2209) [2630](https://github.com/gnolang/gno/pull/2630) are about `MsgNoop`. Hope this will give you a clearer view.

Reply:
Yes, I can guess from the modifications in the gno chain.
But I wanted to check out decoding the proto in gno-js-client.
https://github.com/onbloc/adena-wallet/pull/540/files#diff-ebc97186e74851cee33a3481a356d0f5e231df2e90af1ccaf35ea14cc9fb461aR63-R64

Reply:
> Yes, I can guess from the modifications in the gno chain. But I wanted to check out decoding the proto in gno-js-client. https://github.com/onbloc/adena-wallet/pull/540/files#diff-ebc97186e74851cee33a3481a356d0f5e231df2e90af1ccaf35ea14cc9fb461aR63-R64

@jinoosss 
I think this PR about our changes in gno-js-client may fit.
https://github.com/gnolang/gno-js-client/pull/140
I will update later for further information.